bi_view_editor/security/ir.model.access.csv

```diff
@@ -1,3 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_bve_view_everyone,bve.view,bi_view_editor.model_bve_view,,1,1,1,1
-access_bve_view_line,access_bve_view_line,model_bve_view_line,,1,1,1,1
+access_bve_view_everyone_user,bve.view.user,bi_view_editor.model_bve_view,,0,0,0,0
```

Comment:
Are you sure this is ok, from a funcitonal pov? I'm not very familiar with this module, but like this the "user" has no permissions, not even read permissions (`0,0,0,0`).

From a technical pov, an `ir.access.model` record with 0 permissions is the same as no record, so assuming the functional behaviour is wanted, I'd just remove these lines

Reply:
yep of course, my bad. Thanks @ivantodorovich, I would go with usual user and manager rights. So, `base.group_user` can read, `group_bve_manager` can do CRUD